public/stylesheets/style.css

```diff
@@ -21,3 +21,15 @@
   text-decoration: line-through;
   color: #777;
 }
+
+.button {
```

Comment:
We should explore the use of existing bootstrap button classes so that our styles remain internally consistent.